rosdep/base.yaml

```
@@ -2783,6 +2783,11 @@ libqglviewer-qt4-dev:
     xenial: [libqglviewer-dev-qt4]
     yakkety: [libqglviewer-dev-qt4]
     zesty: [libqglviewer-dev-qt4]
+libqglviewer-qt5-dev:
+  debian: [libqglviewer-dev-qt5]
```

There is a Fedora package for this key as well: https://apps.fedoraproject.org/packages/libQGLViewer-qt5-devel

```suggestion
  debian: [libqglviewer-dev-qt5]
  fedora: [libQGLViewer-qt5-devel]
```